dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskGroupServiceImpl.java

```diff
@@ -189,7 +191,7 @@ public boolean isTheTaskGroupAvailable(int id) {
      */
     @Override
     public Map<String, Object> queryAllTaskGroup(User loginUser, String name, Integer status, int pageNo, int pageSize) {
-        return this.doQuery(loginUser, pageNo, pageSize, loginUser.getUserType().equals(UserType.ADMIN_USER) ? 0 : loginUser.getId(), name, status);
+        return this.doQuery(loginUser, pageNo, pageSize, loginUser.getId(), name, status);
```

Comment:
I think we should not change this since admin should be allowed to query all data.

Reply:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/42627702/178887393-9840f885-8f73-4f6b-9b89-249ce90996d0.png">
I know that the underlying logic still checks the admin